website/build-examples.js

```diff
@@ -21,11 +21,10 @@
  * the Set when it has been bundled.
  */
 
-const createStream = require('fs').createWriteStream
-const glob = require('multi-glob').glob
+const { createWriteStream:createStream, mkdirSync } = require('fs')
```

Comment:
```suggestion
const { createWriteStream: createStream, mkdirSync } = require('fs')
```

maybe we should just name the variable itself `createWriteStream` to keep it clearer, but not a big deal